bin/benchmark/Sources/benchmark/Subcommands/CommitCommands.swift

```diff
@@ -0,0 +1,111 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2022 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+import ArgumentParser
+import SwiftDocC
+
+struct CompareTo: ParsableCommand {
+    @Argument(
+        help: "The baseline 'commit-ish' to compare the current checkout against."
+    )
+    var commitHash: String
```

Comment:
As an extension to this, it might be useful to provide an argument that takes a path to another clone of `docc`, so that you can run comparisons against forks as well.

Reply:
Can you give examples of when we'd want to compare across forks (other than comparing from a PR back to main)?

It's always possible to locally checkout a certain branch from another remote and benchmark against the commit that way. I'm not sure that comparing against forks is common enough that it would be worth automating that step.